### DISCUSSION TITLE
Very Volatile Speed

### DISCUSSION THREAD

**Discussion by kajode** [Chia-Network/chia-blockchain#3252] [Support]
Hey guys,

I got my first plotter up and running now, but the writing keeps on stopping and starting which severly hinders the amount of plots that I can plot.

My specs are: Ubuntu 20.04LTS Chia 1.1.2, i7 6700, 64GB RAM, 2x 1TB NVME

This is what I see using dstat : (the CPU usage seems fine to me with ~70%)

<img width="554" alt="Bildschirmfoto 2021-04-30 um 22 15 16" src="https://user-images.githubusercontent.com/59487189/116753283-23f9d280-aa07-11eb-9de4-446c29f3fd87.png">

Does anyone have an idea what the issue might be?

Thank you in advance!

**Comment by sargonas** [comment]
*auto converting this over to a Discussion to continue community support there, as this does not appear to be an actual development bug at this time.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
